src/plfa/part1/Naturals.lagda.md

```diff
@@ -75,14 +75,14 @@ successor of two; and so on.
 
 #### Exercise `seven` (practice) {#seven}
 
-Write out `7` in longhand.
+Write out `7` in longhand. The suggestion below loads but is, of course, incorrect.
```

Comment:
Why do you want to avoid `?`?

Reply:
Because at this point, the reader hasn't learned how to deal with `?`--the walkthrough of that is a bit later.